bears/vcs/git/GitCommitBear.py

```diff
@@ -60,9 +60,12 @@ def run(self,
             self.err("git:", repr(stderr))
             return
 
-        # git automatically removes trailing whitespaces. Also we need to
-        # remove the last \n printed to align the prompt onto the next line.
-        stdout = stdout.splitlines()[:-1]
+        stdout = stdout.splitlines()
+        if stdout[-1] == "":
+            # Remove the last line if it's empty. git strips trailing empty
+            # lines away and leaves a single one that's empty, but sometimes
+            # the commit message doesn't have a \n.
+            stdout = stdout[:-1]
```

Comment:
git performs the stripping automatically, at least for every commit message I've seen. I think we would do this general-strip-away on demand when something happens again, as I don't know how to reproduce such a case^^